nixos/roles/nix-remote-builder.nix

```diff
@@ -26,14 +26,18 @@ in
       { domain = "*"; item = "nofile"; type = "-"; value = "20480"; }
     ];
 
-    # Give restricted SSH access to the build scheduler
-    users.users.nix-remote-builder.openssh.authorizedKeys.keys = map
-      (key:
-        ''command="nix-daemon --stdio",no-agent-forwarding,no-port-forwarding,no-pty,no-user-rc,no-X11-forwarding ${key}''
-      )
-      cfg.schedulerPublicKeys;
-    users.users.nix-remote-builder.isNormalUser = true;
-    users.users.nix-remote-builder.group = "nogroup";
+    users.users.nix-remote-builder = {
+      isSystemUser = true;
```

Comment:
`isSystemUser` sets the shell to `nologin`. Might be simpler to just keep isNormalUser instead?

Reply:
Normal Users show up in login manager. I have to check what heuristic this is based on. But would be nice if service accounts don't show up in there. Also having a group not equal to users is desirable since this may have other side effects.

Reply:
It's a common pattern to use the normal user flag as a filter for other things in the module system.

Reply:
> It's a common pattern to use the normal user flag as a filter for other things in the module system.

This is sort of what I meant, it depends on the filter, shell and ssh isn't something system users would normally have.